test/integration/testpaths/testpath_volume_snapshot.py

```diff
@@ -0,0 +1,745 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+""" Test cases for VM/Volume snapshot Test Path
+"""
+from nose.plugins.attrib import attr
+from marvin.cloudstackTestCase import cloudstackTestCase, unittest
+from marvin.lib.utils import (cleanup_resources,
+                              random_gen,
+                              format_volume_to_ext3,
+                              is_snapshot_on_nfs,
+                              validateList)
+from marvin.lib.base import (Account,
+                             ServiceOffering,
+                             DiskOffering,
+                             Template,
+                             VirtualMachine,
+                             Snapshot
+                             )
+from marvin.lib.common import (get_domain,
+                               get_zone,
+                               get_template,
+                               list_volumes,
+                               list_snapshots,
+                               list_events,
+                               )
+
+
+import hashlib
+from marvin.sshClient import SshClient
+
+from marvin.codes import PASS
+
+
+def createChecksum(self, virtual_machine, disk, disk_type):
```

Comment:
Can we move this function to common.py file so that it can be used my multiple test cases? It is also used in storage migration test path.
Please move to common.py and make changes in all test paths accordingly.